01/userinfamous/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import LETTER_SCORES, DICTIONARY 
```

Comment:
you can also import from modules that are in the parent directory, so just use `from ..data import`. `.module` means Python is looking for module.py in the current directory and `..module` means Python is looking in the parent directory